docs/source/develop.rst

```diff
@@ -22,6 +22,26 @@ and install it from source::
    cd distributed
    python setup.py install
 
+Using conda, for example::
+
+   git clone git@github.com:{your-fork}/distributed.git
+   cd distributed
+   conda create -y -n distributed python=3.6
+   conda activate distributed
+   pip install -U -r requirements.txt
+   pip install -U -r dev-requirements.txt
+   pip install -e .
```

Comment:
I think the recommended way of using pip now is with `python -m pip ...`. but I can't find where are read that.

Reply:
https://snarky.ca/why-you-should-use-python-m-pip/

Reply:
Done